src/init.cpp

```diff
@@ -5,6 +5,9 @@
 #include "txdb.h"
 #include "walletdb.h"
 #include "bitcoinrpc.h"
+#ifdef REST
```

Comment:
This is probably too generic. Maybe we should call this NEBLIO_REST instead. Conflicts in defs are ugly.

Reply:
agreed